conanfile_py.rst

```diff
@@ -88,13 +90,13 @@ If you are building your project with CMake, edit your ``conanfile.py`` and add
    
       def build(self):
          cmake = CMake(self.settings)
-         self.run('cmake "%s" %s' % (self.conanfile_directory, cmake.command_line))
-         self.run('cmake --build . %s' % cmake.build_config)
+         cmake.configure(self)
```

Comment:
At least at the end of this section I would like to mention that `configure` and `build` are only helpers and you can still use the "command line version".